guides/v2.2/comp-mgr/upgrader/upgrade-start.md

```diff
@@ -37,11 +37,9 @@ To run System Upgrade:
 
 	![Authentication keys entered in the Setup Wizard]({{ site.baseurl }}/common/images/compman_auth-keys.png)
 
-	<div class="bs-callout bs-callout-warning" markdown="1">
-For upgrade or update, you must use the same authentication keys you used to install the Magento software. For example, you *cannot* use {{site.data.var.ce}} authentication keys to update or upgrade Magento EE or vice versa. You also *cannot* use:
-* Another user's authentication keys
-* [Shared account](http://docs.magento.com/m2/ce/user_guide/magento/magento-account-share.html){: target="_blank"} authentication keys
-</div>
+	{:.bs-callout .bs-callout-warning}
+	For upgrade or update, you must use the same authentication keys you used to install the Magento software. For example, you *cannot* use {{site.data.var.ce}} authentication keys to update or upgrade Magento EE or vice versa. You also *cannot* use another user's authentication keys or [Shared account](http://docs.magento.com/m2/ce/user_guide/magento/magento-account-share.html){: target="_blank"} authentication keys
```

Comment:
Delete the extra space between the colon and attribute or delete the attribute completely:

>`{: target="_blank"}`